### PR TITLE
Update README and add authors.txt

### PR DESCRIPTION
**Pull request by tassoskoutlas**
I've updated the README file to reflect the correct workflow for people contributions. I have also added a human.txt to credit everyone who did work or wanted to do some work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,23 +8,32 @@ Slack. You can visit it here: [http://greekdigitalcommunity.com](http://greekdig
...
+git checkout -b issue#XX--name_of_the_issue_you_are_implementing develop
 ```
 
+where issue-XX is the issue and number (e.g. issue-7). Use that branch to
+implement the feature you are working on.
+
 ## Contribute
 
-Once done, please create a pull request.
+Once done, push your branch to the reposiotyr and [create a pull request](https://help.github.com/articles/creating-a-pull-request/).
```
Comment (smosgr) [inline README.md]: *repository

File: README.md [hunk 2]
```diff
@@ -8,23 +8,32 @@ Slack. You can visit it here: [http://greekdigitalcommunity.com](http://greekdig
...
 ```bash
 git clone git@github.com:greekdigitalcommunity/greekdigitalcommunity.github.io.git
 ```
-Set up the remotes:
+
+If you are contributing on an issue create a branch from develop like so:
 ```bash
-git remote set-url upstream git@github.com:greekdigitalcommunity/greekdigitalcommunity.github.io.git
-git remote set-url origin <your_forks_giturl>
+git checkout -b issue#XX--name_of_the_issue_you_are_implementing develop
```
Comment (smosgr) [inline README.md:19]: Remove “develop” word - no need I think

File: index.html [hunk 3]
```diff
@@ -14,8 +14,7 @@
 
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="description" content="">
-    <meta name="author" content="">
+    <meta name="description" content="Greek Digital Community | Η ελληνική ψηφιακή κοινότητα στο Slack">
```
Comment (smosgr) [inline index.html:18]: Ωραίο rewording

**Review by smosgr — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tassoskoutlas) [reply README.md:19]: It ensures that they branch from develop.

**Review by smosgr — APPROVED** [review]

**Pull request merged**